### PR TITLE
CloudWatch: Added AWS WAFV2 metrics

### PR DESCRIPTION
**Pull request by mikkokupsu**
**What this PR does / why we need it**:
Added AWS WAFV2 metrics.

**Which issue(s) this PR fixes**:
No issue.

**Special notes for your reviewer**:
AWS documentation about WAFV2 metrics is not up-to-date but these metrics and dimensions are visible in my AWS account.

### CODE REVIEW

**Comment by mikkokupsu** [comment]
@aknuds1 would you be willing to review this PR or point me to a person whom I could request a review?

**Comment by aknuds1** [comment]
@mikkokupsu We're looking at it, but our main CW expert is on leave. Hopefully we'll come to a conclusion soon.

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 1]
```diff
@@ -126,6 +126,7 @@ func init() {
 		"Rekognition":           {"DetectedFaceCount", "DetectedLabelCount", "ResponseTime", "ServerErrorCount", "SuccessfulRequestCount", "ThrottledCount", "UserErrorCount"},
 		"WAF":                   {"AllowedRequests", "BlockedRequests", "CountedRequests", "DDoSAttackBitsPerSecond", "DDoSAttackPacketsPerSecond", "DDoSAttackRequestsPerSecond", "DDoSDetected", "PassedRequests"},
 		"AWS/Cassandra":         {"AccountMaxReads", "AccountMaxTableLevelReads", "AccountMaxTableLevelWrites", "AccountMaxWrites", "AccountProvisionedReadCapacityUtilization", "AccountProvisionedWriteCapacityUtilization", "ConditionalCheckFailedRequests", "ConsumedReadCapacityUnits", "ConsumedWriteCapacityUnits", "MaxProvisionedTableReadCapacityUtilization", "MaxProvisionedTableWriteCapacityUtilization", "ReturnedItemCount", "ReturnedItemCountBySelect", "SuccessfulRequestCount", "SuccessfulRequestLatency", "SystemErrors", "UserErrors"},
+		"AWS/WAFV2":             {"AllowedRequest", "BlockedRequests", "CountedRequests"},
```
Comment (aknuds1) [inline pkg/tsdb/cloudwatch/metric_find_query.go:129]: I think this should be [AllowedRequests](https://docs.aws.amazon.com/waf/latest/developerguide/monitoring-cloudwatch.html#waf-metrics)? Also, should we include PassedRequests?
```suggestion
		"AWS/WAFV2":             {"AllowedRequests", "BlockedRequests", "CountedRequests", "PassedRequests"},
```

**Review by papagian — CHANGES_REQUESTED** [review]
Thank you for your contribution!

According to this [document](https://docs.aws.amazon.com/waf/latest/APIReference/API_VisibilityConfig.html) I understand that the metrics/dimensions should be the same with [`AWS WAF`](https://docs.aws.amazon.com/waf/latest/developerguide/monitoring-cloudwatch.html#waf-metrics), therefore I think that `PassedRequests` should be included in the metrics and `RuleGroup` in the dimensions.

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 2]
```diff
@@ -213,6 +214,7 @@ func init() {
 		"Rekognition":           {},
 		"WAF":                   {"Region", "Rule", "RuleGroup", "WebACL"},
 		"AWS/Cassandra":         {"Keyspace", "Operation", "TableName"},
+		"AWS/WAFV2":             {"Region", "Rule", "WebACL"},
```
Comment (aknuds1) [inline pkg/tsdb/cloudwatch/metric_find_query.go:217]: Looks to me as if [RuleGroup](https://docs.aws.amazon.com/waf/latest/developerguide/monitoring-cloudwatch.html#waf-metrics) is among the dimensions:
```suggestion
		"AWS/WAFV2":             {"Region", "Rule", "WebACL", "RuleGroup"},
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]

**Comment by aknuds1** [comment]
@mikkokupsu I've resolved the merge conflicts locally, do you want me to push the result to your fork?

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
@papagian It look OK to you?

**Review by papagian — APPROVED** [review]
LGTM!

**Pull request merged**